index.html

```diff
@@ -1,120 +1,90 @@
 <!doctype html>
 <html>
-	
+
 	<head>
-		<link rel="stylesheet" href="stylesheet.css"/>
-		<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+		<link rel="shortcut icon" href="./images/favicon.ico" type="image/x-icon"/>
 		<title>Smelt: Batch Package Manager</title>
+		<style>
+		code {
+			color:#3498db;
+		}
+		</style>
 	</head>
-	
-	
+
+
 <body>
-		
-	<div class="navbar">
-	<br/>
-        <h1>Smelt: Batch Package Manager</h1>
-	<br/>
-        </div>
-	
-	<br/>
-	<br/>
-	<br/>
-	
-	<div class="boxes">
-	<img src="windows.png">
-	<h2>Windows Installation</h2>
-	<h3>Install this package manager</h3>
-	
-		
-	<ol>
-	<li>Download <a href="https://smelt-modding.github.io/smelt/setup.bat"><code>setup.bat</code></a>.</li>
-	<li>Double-click it.</li>
-	<li>You're done! There will be:<ul>
-	<li>two new folders: 
-	<br/><code>extension</code> and <code>logs</code></li>
-	<li>and five new files:</l1>
-	<br/>
-	<code>sbpm.bat, install.bat, uninstall.bat, update.cmd, and reinstall.bat</code>
-	</ul>
-	</li>
-	</ol>
-	</div>
 
-	
-	<div class="boxes">
-	<h3>To install commands pre-downloaded from other sources:</h3>
-	<ol>
-	<li>Move the downloaded commands into the <code>extensions</code> folder.</li>
-	<li>Double-click <code>install.bat</code></li>
```

Comment:
Looking at the preview, `<code>` tags do seem to be monospaced, but blue highlight isn't really good - I'd prefer default font color and some sort of background color - preferably with a little padding.